include/dba.php

```diff
@@ -656,6 +672,8 @@ public static function exists($table, $condition) {
 	/**
 	 * @brief Fetches the first row
 	 *
+	 * Please use dba::select or dba::select whenever this is possible.
```

Comment:
is it really `dba::select` or `dba::select`?

Reply:
Nitpicker ;-)